client/src/components/RelatedProducts/Cards/Comparison.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
 import styled from 'styled-components';
+import sampleFeatures from '../sampledata/sampleFeatures';
+import Features from './Features';
 
-const Testing = styled.div`
-  border-style: solid;
-  height: 500px;
-  width: 500px;
-`;
+// should get an array with two arrays of features from each item
 function Comparison() {
```

Comment:
Love that the comparison is working! If you want it to pop outside of the card, you can set the `position` of the outer card to be `relative` and then the `position` of the popped out modal to be `absolute`. then you can set the `top` and `left` properties to be negative numbers so that they float outside of the modal!